app/controllers/api/v3/encounters_controller.rb

```diff
@@ -11,6 +11,8 @@ def sync_to_user
   end
 
   def generate_id
+    raise ActionController::RoutingError.new("Not Found") unless Flipper.enabled?("generate_encounter_id_endpoint")
```

Comment:
We can't use Flipper in the routes file (it's not initialized yet), so we resort to manually returning a "404 sorry" in the controller here.